plugins/services/src/js/components/modals/ServiceRootGroupModal/index.tsx

```diff
@@ -196,8 +243,36 @@ class ServiceRootGroupModal extends React.Component<
     });
   }
 
+  getGroupFormData(): void {
+    const { id } = this.props;
+    if (!!id) {
+      getGroup(id)
+        .pipe(take(1))
+        .subscribe({
+          next: groupData => {
+            const data = groupFormDataFromGraphql(groupData.data.group);
+            this.setState({
+              data,
+              originalData: JSON.parse(JSON.stringify(data))
+            });
+          }
+        });
+    }
+    this.setState({
```

Comment:
should this be in an else? in edit mode we wouldn't want data to be set until the query has finished?

Reply:
I am not sure, but I think we don't even need it.